.github/actions/next-internal-version/action.yaml

```diff
@@ -0,0 +1,29 @@
+name: "Get Next Internal Release Version"
+description: "Determines if the published image should include `latest` tags"
```

Comment:
Does this determine if `latest` should be used, what the next version is, both? This description doesn't appear to match the `outputs` on the surface.